### ISSUE TITLE
In-line row editing - Reset row not working 

### ISSUE THREAD

**Issue by razazaidisd** [esvit/ng-table#884]
- Click on any edit Row (except first row)
- Then click on the Cancel (X) button, IT WILL RESET THE CURRENT ROW to the FIRST ROW OF THE TABLE.

http://ng-table.com/#/editing/demo-inline

**Comment by anuja-joshi** [comment]
@razazaidisd yes there is an issue with demo. Function `resetRow` is returning incorrect row because of `_.findWhere`.
I tried to resolve issue, here is link to the [codepen](http://codepen.io/anuja-joshi/pen/ZpBRpE) for updated code.
This issue is same as [issue #912](https://github.com/esvit/ng-table/issues/912).
